xml/admin_monitoring_alerting.xml

```diff
@@ -1,1058 +1,374 @@
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>LABELVALUE</replaceable></term>
-     <listitem>
+  <para>
+    We recommend that the <systemitem class="daemon">Node exporter</systemitem>
+    will be deployed on all nodes. This can be done using the
+    <filename>monitoring.yaml</filename> file with the
+    <literal>node-exporter</literal> service type. See <xref
+    linkend="monitoring-stack-deploy" /> for more information on deploying
+    services.
+  </para>
+
+  <sect1 xml:id="monitoring-stack-deploy">
+    <title>Deploying Monitoring With &cephadm;</title>
+    <para>
+      To deploy the monitoring stack you need to either use a YAML configuration
+      file or the CLI. Whichever method you will use, you need to ensure the
+      &prometheusmgr; is enabled.
+    </para>
+    <para>
+      The &prometheusmgr; needs to be enabled using the following command:
+    </para>
+    <screen>&prompt.cephuser;ceph mgr module enable prometheus</screen>
+    <note>
       <para>
-       A string of Unicode characters.
+        Ensure this command is run before &prometheus; is deployed. If the
+        <command>ceph mgr module enable prometheus</command> was not run before
+        deployment, you must redeploy &prometheus; to update Prometheus'
+        configuration:
       </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>FILEPATH</replaceable></term>
-     <listitem>
+      <screen>&prompt.cephuser;ceph orch redeploy prometheus</screen>
+    </note>
+    <important>
       <para>
-       A valid path in the current working directory.
+        &cephadm; handles the configuration of &prometheus;, &grafana;, and the
+        &alertmanager; automatically.
       </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>BOOLEAN</replaceable></term>
-     <listitem>
       <para>
-       A Boolean that can take the values 'true' or 'false'.
+        The &prometheusmgr; can be configured, though. See <xref
+        linkend="monitoring-cephadm-config" /> for more options.
       </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>STRING</replaceable></term>
-     <listitem>
+    </important>
+    <sect2 xml:id="monitoring-service-spec">
+      <title>Deploying Using a Service and Placement Specification</title>
       <para>
-       A regular string.
+        &cephadm; also uses a <filename>.yaml</filename> configuration file to
+        deploy the relevant services, see <xref
+        linkend="cephadm-service-and-placement-specs" /> for more information on
+        creating service and placement specifications.
       </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>SECRET</replaceable></term>
-     <listitem>
       <para>
-       A regular string that is a secret, for example a password.
+        To apply a configuration file and thereby deploy the configured
```

Comment:
```suggestion
        To apply a configuration file, and thereby deploy the configured
```

Reply:
I still don't like "thereby" here, though.

Would it also be valid to say:

"To configure and deploy the services with a configuration file, create a file called..."
...?